platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ProblemReportingCrossProjectModelAccess.kt

```diff
@@ -930,38 +539,13 @@ class ProblemReportingCrossProjectModelAccess(
             shouldNotBeUsed()
         }
 
-        override fun getTasks(): TaskContainerInternal {
-            onAccess("tasks")
-            return delegate.tasks
-        }
-
         override fun getBuildScriptSource(): ScriptSource {
             shouldNotBeUsed()
         }
 
-        override fun project(path: String): ProjectInternal {
-            return delegate.project(referrer, path)
-        }
-
-        override fun project(referrer: ProjectInternal, path: String): ProjectInternal {
-            return delegate.project(referrer, path)
-        }
-
-        override fun findProject(path: String): ProjectInternal? {
-            return delegate.findProject(referrer, path)
-        }
-
-        override fun findProject(referrer: ProjectInternal, path: String): ProjectInternal? {
-            return delegate.findProject(referrer, path)
-        }
-
-        override fun getInheritedScope(): DynamicObject {
-            return delegate.inheritedScope
-        }
-
         override fun getGradle(): GradleInternal {
```

Comment:
✨  We should group the `shouldNotBeUsed()` methods  together, and move methods we override before them.